nasa/earthdata-download
# EDD-13: As a download app user, I want to be able to see a list of the individual files I'm downloading

# Overview

### What is the feature?

Adding Granule level view and controls

### What is the Solution?

Adding new components and a secondary interval polling layer with database calls to
retrieve data from the `files` table then update the view

### What areas of the application does this impact?

main functions from ipc channels, adds a new page, adds components for the view on the pages

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. Start a download on EDD click on one of the downloads to open up the fileDownloads page
2. Ensure that progress is fulfilling with remaining time decreasing and bytes progress incrementing
3. Ensure that the fileDownloads are completing on this view
4. Ensure that if we click on the `Hide Completed` button will hide all of the completed file downloads

5. Ensure that we can open files using the menu items, restart downloads after they are completed. and that the copy file path gives us a useful filepath by using `open <paste-here> which should open up the file

5. Ensure we can return to the previous location with the back to downloads button

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
